Tasks/AzureRmWebAppDeployment/XMLSubstituter.ps1

```diff
@@ -0,0 +1,74 @@
+﻿Param(
+  [Parameter(mandatory=$true)][string]$xmlFilePath,
+  [Parameter(mandatory=$true)][string]$tag,
+  [string]$conditionalAttributeKey,
+  [string]$conditionalAttributeValue,
+  [Parameter(mandatory=$true)][string]$targetKey,
+  [Parameter(mandatory=$true)][string]$targetValue
+)
+
+Write-Verbose "xmlFile = $xmlFilePath" -Verbose
```

Comment:
-verbose is not recommended now. if we set system.debug true all these statements are printed without -verbose verb and if you want to specify in console window then use write-host get-localized...refer checked in task in master